prdoc/pr_9480.prdoc

```diff
@@ -0,0 +1,85 @@
+title: Lazy decode block extrinsics
+doc:
+- audience: Runtime Dev
+  description: |-
+    Related to https://github.com/paritytech/polkadot-sdk/issues/4255
+
+    Addresses https://github.com/paritytech/polkadot-sdk/pull/7902#pullrequestreview-3079855302
+
+    This PR enables us to lazily decode the block extrinsics when needed.
```

Comment:
Maybe we can provide a guide about how to update and what is breaking.

Reply:
Good point ! Done.